app/aws-lsp-json-runtimes/src/tests/jsonServerCFInteg.test.ts

```diff
@@ -12,41 +12,51 @@ import {
     TEXT_TO_DIAGNOSE_JSON,
     TEXT_TO_FORMAT_JSON,
     TEXT_TO_HOVER_JSON,
+    DIAGNOSTICS_RESPONSE_JSON_CUSTOM,
+    HOVER_JSON_CUSTOMIZED,
 } from './testUtilsCF'
 
-describe('Test JsonServer with CloudFormation schema', () => {
+async function createLSPServer(runtimeFile: string) {
     const rootPath = path.resolve(__dirname)
     let process: ChildProcessWithoutNullStreams
-    let endpoint: JSONRPCEndpoint
-    let client: LspClient
-    const runtimeFile = path.join(__dirname, '../../', 'build', 'aws-lsp-json-standalone.js')
 
-    before(async () => {
-        // start the LSP server
-        process = spawn('node', [runtimeFile, '--stdio'], {
-            shell: true,
-            stdio: 'pipe',
-        })
+    // Start the LSP server
+    // eslint-disable-next-line prefer-const
+    process = spawn('node', [runtimeFile, '--stdio'], {
+        shell: true,
+        stdio: 'pipe',
+    })
 
-        // create an RPC endpoint for the process
-        endpoint = new JSONRPCEndpoint(process.stdin, process.stdout)
+    // Create an RPC endpoint for the process
+    const endpoint = new JSONRPCEndpoint(process.stdin, process.stdout)
 
-        // create the LSP client
-        client = new LspClient(endpoint)
+    // Create the LSP client
+    const client = new LspClient(endpoint)
 
-        const result = await client.initialize({
-            processId: process.pid ?? null,
-            capabilities: {},
-            workspaceFolders: [
-                {
-                    name: 'workspace',
-                    uri: pathToFileURL(rootPath).href,
-                },
-            ],
-            rootUri: null,
-        })
+    // Initialize the LSP client
+    await client.initialize({
+        processId: process.pid ?? null,
+        capabilities: {},
+        workspaceFolders: [
+            {
+                name: 'workspace',
+                uri: pathToFileURL(rootPath).href,
+            },
+        ],
+        rootUri: null,
+    })
 
-        expect(result.capabilities).to.exist
+    return { client, endpoint, process }
+}
+
+describe('Test JsonServer with CloudFormation schema', () => {
+    let client: LspClient
+    let endpoint: JSONRPCEndpoint
+    let process: ChildProcessWithoutNullStreams
+    const runtimeFile = path.join(__dirname, '../../', 'build', 'aws-lsp-json-standalone.js')
+
+    before(async () => {
+        ;({ client, endpoint, process } = await createLSPServer(runtimeFile))
```

Comment:
Is the semicolon (`;`) needed?

```suggestion
        ({ client, endpoint, process } = await createLSPServer(runtimeFile))
```

Reply:
No - this got added automatically added by one of the formatters - I will disable it and revert to the right way.

Reply:
If it's auto-formatted it's fine. It's just not frequent to see it and in this specific case it doesn't seem required. But no harm in keeping it.